ProofPoint/proofpoint-pod/tests/test_msgparts.json

```diff
@@ -16,6 +16,7 @@
         "email"
       ],
       "dataset": "msgParts",
+      "action": "inline",
```

Comment:
The JSON document in the field `message` should be updated too with the new property `disposition`

Reply:
Ok i have understand, thank you 🙏.